tests/data/sample_pointing_videos_161215_15_34_21.sig2

```diff
@@ -453,861 +456,1205 @@ Generating signature for sample_pointing_videos_161215_15_34_21.nwb
-/general/optophysiology/Zstack0041/green/emission_lambda: dtype=float32 shape=() val="nan"
-/general/optophysiology/Zstack0041/imaging_rate: dtype=float32 shape=() val="348.7419135"
-/general/optophysiology/Zstack0041/indicator: dtype=object shape=() val="Calcium indicator."
-/general/optophysiology/Zstack0041/location: dtype=object shape=() val="Anatomy gross description o...0x7a562276"
-/general/optophysiology/Zstack0041/manifold: dtype=float32 shape=(512, 512, 3) val="0x6407104f"
-	@conversion: dtype=float64 shape=() val="1000000"
-	@unit: type=str val="metre"
-/general/optophysiology/Zstack0041/red/description: dtype=object shape=() val="Red channel, typically used...0x49f80f71"
-/general/optophysiology/Zstack0041/red/emission_lambda: dtype=float32 shape=() val="nan"
-/general/optophysiology/Zstack0041/reference_frame: dtype=object shape=() val="TODO: In lab book (partly?)"
-/general/optophysiology/Zstack0042/description: dtype=object shape=() val="Reference Z stack"
-/general/optophysiology/Zstack0042/device -> /general/devices/AOL_microscope
-/general/optophysiology/Zstack0042/excitation_lambda: dtype=float32 shape=() val="nan"
-/general/optophysiology/Zstack0042/green/description: dtype=object shape=() val="Green channel, typically us...0xb22811b2"
-/general/optophysiology/Zstack0042/green/emission_lambda: dtype=float32 shape=() val="nan"
-/general/optophysiology/Zstack0042/imaging_rate: dtype=float32 shape=() val="348.7419135"
-/general/optophysiology/Zstack0042/indicator: dtype=object shape=() val="Calcium indicator."
-/general/optophysiology/Zstack0042/location: dtype=object shape=() val="Anatomy gross description o...0x7a562276"
-/general/optophysiology/Zstack0042/manifold: dtype=float32 shape=(512, 512, 3) val="0xa69412a7"
-	@conversion: dtype=float64 shape=() val="1000000"
-	@unit: type=str val="metre"
-/general/optophysiology/Zstack0042/red/description: dtype=object shape=() val="Red channel, typically used...0x49f80f71"
-/general/optophysiology/Zstack0042/red/emission_lambda: dtype=float32 shape=() val="nan"
-/general/optophysiology/Zstack0042/reference_frame: dtype=object shape=() val="TODO: In lab book (partly?)"
-/general/optophysiology/Zstack0043/description: dtype=object shape=() val="Reference Z stack"
-/general/optophysiology/Zstack0043/device -> /general/devices/AOL_microscope
-/general/optophysiology/Zstack0043/excitation_lambda: dtype=float32 shape=() val="nan"
-/general/optophysiology/Zstack0043/green/description: dtype=object shape=() val="Green channel, typically us...0xb22811b2"
-/general/optophysiology/Zstack0043/green/emission_lambda: dtype=float32 shape=() val="nan"
-/general/optophysiology/Zstack0043/imaging_rate: dtype=float32 shape=() val="348.7419135"
-/general/optophysiology/Zstack0043/indicator: dtype=object shape=() val="Calcium indicator."
-/general/optophysiology/Zstack0043/location: dtype=object shape=() val="Anatomy gross desc
```

Comment:
Should we be ignoring this field?

Reply:
Which field is that? The ImagingPlane datasets are currently duplicated when we have two channels (#14) but all these fields are still present, as far as I can see.

Reply:
Huh, that didn't put the comment on the right line! I meant the silverlab API version.

Reply:
Hm, possibly. I don't imagine it will change often, but that also means it won't be very helpful in figuring out how a signature was generated. I'll add it to the next batch of changes (#28) so we don't need to regenerate and rebase all the signatures generated since this...

Reply:
Good point. Perhaps we do want to keep it in then. It's an easy one to update if we see a change!